Modules/Microsoft365DSC/DSCResources/MSFT_EXOSmtpDaneInbound/MSFT_EXOSmtpDaneInbound.psm1

```diff
@@ -0,0 +1,343 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $DomainName,
+
+        [Parameter()]
+        [ValidateSet('Present', 'Absent')]
+        [System.String]
+        $Ensure,
```

Comment:
I wouldn't use the Ensure parameter here. Normally that would be used for objects that can be created or removed, like policies for example. In this case it is for a status, so I would suggest renaming this parameter to SmtpDaneStatus, similar to the cmdlet parameter name.